.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
please exclude IDE-specific files from the commit. Feel free to add `.vscode/` to the root `.gitignore` file.